README.md

```diff
@@ -62,33 +62,39 @@ Thank you to all those who have translated asbplayer:
 If you are a non-English native, and would like to help translate asbplayer, please contact me on [Discord](https://discord.gg/ad7VAQru7m).
 
 ## Getting Started
+> [!NOTE]  
+> asbplayer can either be used purely as a tool for subtitle control or as a flashcard creation tool. If you want to strictly learn how to use asbplayer's subtitle features and are not interested in sentence or vocabulary mining, you can read about them under the [Subtitle setup and features](#subtitle-setup-and-features) header.
 
-First, see if you can get started with one of the [community guides](#community-guides).
+First, see if you can get started by following one of the [community guides](#community-guides).
 
-Otherwise, the following steps will work for any language:
+Otherwise, the following steps for setting up automated Anki flashcards should work for any language:
 
-1. Install a dictionary for your target language that allows you to do instant lookups. Popular ones are [Yomitan](https://chromewebstore.google.com/detail/yomitan/likgccmbimhjbgkjambclfkhldnlhbnn) for Japanese and [VocabSieve](https://github.com/FreeLanguageTools/vocabsieve) for European languages.
+1. Install and set up a dictionary tool for your target language that allows you to do instant lookups. Popular ones are [Yomitan](https://chromewebstore.google.com/detail/yomitan/likgccmbimhjbgkjambclfkhldnlhbnn) for Japanese and [VocabSieve](https://github.com/FreeLanguageTools/vocabsieve) for European languages.
 2. Install [Anki](https://apps.ankiweb.net/), and create a deck and note type.
-3. Install the [AnkiConnect](https://ankiweb.net/shared/info/2055492159) plugin for Anki.
-4. [Configure](https://killergerbah.github.io/asbplayer/?view=settings) asbplayer to create cards via AnkiConnect using your deck and note type.
-
-5. Enhance a video using asbplayer and subtitle files.
+   - *Note: In order to use asbplayer's flashcard audio and screenshot capabilities, you will need to set the deck up accordingly. You can learn how to do this from one of the [community guides](#community-guides).*

```

Comment:
This can be shorter:
"Use one of the community guides to learn how to set up an Anki deck."

We are already telling the user to setup an Anki deck, so I don't think we need to repeat that instruction.